app/uk/gov/hmrc/pensionschemereturn/models/nonsipp/assets/Bonds.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pensionschemereturn.models.nonsipp.assets
+
+import play.api.libs.json.{Json, OFormat}
+
+import java.time.LocalDate
+
+case class Bonds(
+  bondsWereAdded: Boolean,
+  bondsWereDisposed: Boolean,
+  bondTransactions: Seq[BondTransactions]
```

Comment:
Same question as on the FE code, does this need to be Option[Seq[BondTransactions]] ?

Reply:
```
case class EtmpBonds(
  recordVersion: Option[String],
  bondsWereAdded: String,
  bondsWereDisposed: String,
  noOfTransactions: Option[Int],
  bondTransactions: Option[Seq[EtmpBondTransactions]]
)
```
This is handled by `EtmpBonds` case class. `Bonds` are representing FE logic.